wrappers/node/src/api/common.ts

```diff
@@ -125,6 +125,7 @@ export enum HolderStateType {
 export enum IssuerStateType {
   Initial = 0,
   ProposalReceived = 1,
+  OfferSet = 12,
```

Comment:
Please, make `OfferSet` value 2 and adjust the other values (and the tests if necessary)

Reply:
updated